advocacy_docs/edb-postgres-ai/console/getstarted/index.mdx

```diff
@@ -0,0 +1,82 @@
+
+![Console Overview](./images/placeholder-overview.png)
+
+## Exploring the Overview
+
+In the body of the overview page, you can see two summaries of the [__Project__](#projects) and [__Estate__](#estate) views. 
+
+You can directly access the full __Project__ and __Estate__ views by selecting the **Projects** or **Estate** buttons next to the EDB logo. Alson
+
+## Projects
+
+To create and manage resources, you’ll need a project. A project associates organizationally related resources together which enables simpler administration and access.
+
+In the overview's view of projects, you can see the name of each project, an option to add tags, the number of clusters within the project and the number of users who have access to that project.
+
+Selecting **View Projects** takes you to the Projects view.
+
+If you are an owner of the organization, you can create new projects by selecting the **Create New Project** button
+
+## Estate	
+
+The Estate view is your everything view of every resource - clusters, hosted, and managed, self-managed, analytics lakehouses, and managed storage locations - in every project. It cuts through the Projects demarcation to give a single unified view of all your resources. 
+
+Rather than being broken down into projects, the Estate overview is broken down into types of resources.
+
+Each resource shows its type, the number of instances of that resource and, on the right of the pane, a graphical representation or breakdown of those instances.
+
+### EDB Postgres AI Clusters
+
+This is any Postgres cluster created by and managed by EDB Postgres AI. EDB Postgres AI can host these clusters on any supported cloud service provider (AWS, Azure, GCP), or it can host them using  by your own account on any cloud service provider. 
+
+Also included in this view are EDB Postgres AI Lakehouse analytics clusters. 
+
+The graphical view shows a color coded snapshot of the clusters, along with a textual description of which cloud service providers are in use and how many are Cloud Service Hosted by EDB or Managed but not hosted by EDB.
+
+The **Create New** button allows you to create a new Database cluster or Lakehouse analytics cluster in any of your projects. First, select between creating a Database cluster or a  Lakehouse analytics node. If you only have one project, the creation operation will automatically use it. If you have more than one project, you will see a menu of available projects to create your cluster in.
+
+Selecting the **EDB Postgres AI Clusters** title takes you to the __EDB Postgres AI Cluster__ pane of the full Estate view.
+
+### Self Managed Postgres
+
+Self-managed Postgres clusters installed both on-premises and in the cloud can be integrated into the EDB Estate view by using an agent. The agent will collect metrics from an associated cluster and feed it to the EDB Estate. It’s in this pane that that information is rendered.
```

Comment:
Again the thing about Estate... man this naming is troublesome.

Reply:
```suggestion
Self-managed Postgres clusters installed both on-premises and in the cloud can be integrated into your EDB Estate view by using an agent. The agent will collect metrics from an associated cluster and feed it to the EDB Estate. It’s in this pane that that information is rendered.
```